explore/src/main/scala/explore/itc/ItcGraphQuerier.scala

```diff
@@ -98,10 +98,16 @@ case class ItcProps(
 
   val targets: List[ItcTarget] = itcTargets.foldMap(_.toList)
 
-  private val queryProps: List[Option[?]] =
-    List(itcTargets, finalConfig, wavelength, instrumentConfig, signalToNoise)
+  private val queryProps =
+    (wavelength,
+     signalToNoise,
+     signalToNoiseAt,
+     constraints.some,
+     itcTargets,
```

Comment:
Is it nott enough to query only for target ids?

Reply:
Does the itc know the target ids?

Reply:
This would be a bigger change. I'm going to merge, but we can continue discussing.

Reply:
If I recall this is used only to tell  if the properties have changed, and using the ids would bee more efficient that comparing the whole target

Reply:
The ItcTargets, with the gaia bands removed, are passed to `ItcClient.spectroscopyIntegrationTimeAndGraphs`. So, I assume the itc needs the target information?

Reply:
I see, for reusability would the `id` be enough?

Reply:
I don't think so. If a target is edited, it keeps the same id but the itc results could be affected.